src/pkg/packager/interactive.go

```diff
@@ -26,7 +26,7 @@ func (p *Packager) confirmAction(userMessage string, sbomViewFiles []string) (co
 	if len(sbomViewFiles) > 0 {
 		cwd, _ := os.Getwd()
 		link := filepath.Join(cwd, config.ZarfSBOMDir, filepath.Base(sbomViewFiles[0]))
-		msg := fmt.Sprintf("This package has %d images with software bill-of-materials (SBOM) included. You can view them now in the zarf-sbom folder in this directory or to go directly to one, open this in your browser: %s\n * This directory will be removed after package deployment.", len(sbomViewFiles), link)
+		msg := fmt.Sprintf("This package has %d artifacts with software bill-of-materials (SBOM) included. You can view them now in the zarf-sbom folder in this directory or to go directly to one, open this in your browser: %s\n * This directory will be removed after package deployment.", len(sbomViewFiles), link)
```

Comment:
Based on your updated screenshot, it doesn't look like the `\n` here is actually creating a new line in the logged message. 

Thoughts on logging that note in its own `message.Note(...)` call?

Reply:
`message.Note` does auto line breaks, FYI

Reply:
I do like it better this way though:

![image](https://user-images.githubusercontent.com/3977569/207698808-bf9cd592-d6e2-4be9-8b0b-ddda8fd6d8ee.png)